scenarios/aws/microVMs/microvms/run.go

```diff
@@ -92,6 +92,10 @@ func (i *Instance) GetIP() pulumi.StringOutput {
 	return i.instance.GetIP()
 }
 
+const metalUserData = `#!/bin/bash
```

Comment:
Can you please add a comment like this:

`User data shell scripts must start with the #! characters and the path to the interpreter you want to read the script (commonly /bin/bash).`